conda-recipe/meta.yaml

```
@@ -23,7 +23,6 @@ requirements:
   - python >=3.9
   - pyqt >=5
   - pyca =3.2.0
```

This line is also pretty suspicious, it should maybe be >= instead of fixed
```suggestion
  - pyca >=3.2.0
```